bp_me/src/include/v/bp_me_cce_mem_if.vh

```diff
@@ -61,7 +61,7 @@ typedef enum logic [3:0]
  * speculative is set if the request was issued speculatively by the CCE
  */
 
-`define declare_bp_cce_mem_msg_payload_s(lce_id_width_mp, lce_assoc_mp) \
+`define declare_bp_cce_mem_msg_payload_s(lce_id_width_mp, lce_assoc_mp, name_mp) \
```

Comment:
Probably rename cce_mem -> mem throughout, since this memory format is not specific to CCE.  Thoughts @muwyse ?

Reply:
Agreed. bp_mem_if perhaps? Or do we want bp_me_mem_if? I vote for the first option and that this header move to bp_common/src/include/

Reply:
>I vote for the first option and that this header move to bp_common/src/include/

Works for me!